docs/developer-docs/defi/icrc-1/token-quickstart.mdx

```diff
@@ -0,0 +1,116 @@
+---
+keywords: [intermediate, defi, overview, tokens, deploy a token, create a token, icrc1]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Token quickstart
+
+<MarkdownChipRow labels={["Intermediate"  ]} />
+
+## Overview
+
+Custom tokens can be created and deployed on ICP using the ICRC-1 token standard. This standard defines a set of general functionalities that all ledgers must adhere to. ICRC-1 is designed to create a universally accepted standard for creating and recording tokens and their transactions on ICP. Any tokens and their corresponding ledgers must fulfill all requirements of the standard.
+
+There are several extensions of the ICRC-1 standard that extend functionality, however, not all ICRC standards must be an extension of the ICRC-1 standard.
+
+This guide will provide a quickstart on how you can create and deploy your own custom token on ICP using the ICRC-1 standard.
+
+## Create a new project
+
+First, download `dfx` and start the local replica:
+
+```
+sh -ci "$(curl -fsSL https://internetcomputer.org/install.sh)" // Download and install `dfx`.
+dfx start --clean --background // Start the local replica.
+```
+
+Then create a new project:
+
+```
+dfx new new_token
+```
+
+## Setup the ledger canister
+
+Download and install the latest Wasm for the ICRC-1 ledger canister:
+
+```
+curl -o download_latest_icrc1_ledger.sh "https://raw.githubusercontent.com/dfinity/ic/326df23607fc8280a047daba2d8462f1dfc57466/rs/rosetta-api/scripts/download_latest_icrc1_ledger.sh"
+chmod +x download_latest_icrc1_ledger.sh
+./download_latest_icrc1_ledger.sh
+```
+
+## Edit `dfx.json`
+
+Next, edit the project's `dfx.json` to define the Candid and Wasm files for the ICRC-1 ledger canister:
+
+```json
+  "canisters": {
+    "icrc1_ledger_canister": {
+      "type": "custom",
+        "candid": icrc1_ledger.did,
+        "wasm" : icrc1_ledger.wasm.gz,
```

Comment:
@jessiemongeon1 should these be wrapped in quotes too if this is JSON format?

Reply:
Good catch, thanks!